app/models/manageiq/providers/openstack/network_manager/cloud_network.rb

```diff
@@ -1,15 +1,23 @@
 class ManageIQ::Providers::Openstack::NetworkManager::CloudNetwork < ::CloudNetwork
+  include ProviderObjectMixin
+  include AsyncDeleteMixin
+  include SupportsFeatureMixin
+
   require_nested :Private
   require_nested :Public
 
+  supports :update_cloud_network
+  supports :create_cloud_network
+  supports :delete_cloud_network
+
   def self.remapping(options)
     new_options = options.dup
     new_options[:router_external] = options[:external_facing] if options[:external_facing]
     new_options.delete(:external_facing)
     new_options
   end
 
-  def self.raw_create_network(ext_management_system, options)
+  def self.create_cloud_network(ext_management_system, options)
```

Comment:
This should probably keep the raw/unraw split like we have elsewhere for consistency.  The raw/unraw split allows us to have one method that does the actual action (raw) and a separate method to allow us to wrap that operation in policies.